README.md

```diff
@@ -41,6 +42,9 @@ merch_week.end_of_month         # <Date: 2014-01-04>
 merch_week.start_of_year        # <Date: 2013-02-03>
 merch_week.end_of_year          # <Date: 2014-02-01>
 
+merch_week.calendar             # <MerchCalendar::RetailCalendar> (the calendar we're using)
```

Comment:
I don't know that I see a benefit to exposing this on the public interface of the class, but it doesn't really hurt anything.

Reply:
What should I do? Should I delete the comment?

Reply:
If it were me, I probably woudn't have added the `attr_accessor` to the class.  But I think it's fine to keep what you have.